package.json

```diff
@@ -2,7 +2,8 @@
   "name": "peer-id",
   "version": "0.6.4",
   "description": "IPFS Peer Id implementation in Node.js",
-  "main": "src/index.js",
+  "main": "lib/index.js",
+  "jsnext": "src/index.js",
```

Comment:
for a moment, I thought I actually had never fixed anything. But updated here and all still seems good:

https://github.com/diasdavid/js-peer-id/pull/18